.circleci/config.yml

```diff
@@ -31,6 +43,62 @@ jobs:
       - run: yarn test --ci --runInBand
       - run: bash <(curl -s https://codecov.io/bash)
 
+  install-e2e-dependencies:
+    <<: *e2e-defaults
+    steps:
+      - attach_workspace:
+          at: ~/repo
+      - run:
+          name: prepare backend
+          command: |
+            set -x
+            export LC_ALL=C.UTF-8
+            export LANG=C.UTF-8
+            apt-get update
+            apt-get install -y python3 python3-dev virtualenv
+            git clone https://github.com/yunity/karrot-backend
+            cd karrot-backend
+            virtualenv -p python3 env
+            env/bin/pip install --upgrade pip
+            env/bin/pip install -r requirements-dev.txt
+            env/bin/pip-sync requirements*.txt
+            cp config/local_settings.py.ci config/local_settings.py
+      - persist_to_workspace:
+          root: ~/repo
+          paths:
+            - karrot-backend/*
+
+  e2e-test:
+    <<: *e2e-defaults
+    steps:
+      - attach_workspace:
+          at: ~/repo
+      - run:
+          name: run backend
+          command: |
+            set -x
+            export LC_ALL=C.UTF-8
+            export LANG=C.UTF-8
+            apt-get update
+            apt-get install -y python3 python3-dev virtualenv
+            cd karrot-backend
+            env/bin/python manage.py migrate
+            env/bin/python manage.py runserver
+          background: true
+      - run:
+          name: run frontend
+          command: yarn serve
+          background: true
+      - run: wget --retry-connrefused -T 30 http://localhost:8080
+      - run: wget --retry-connrefused -T 30 http://localhost:8000/api/groups-info/
+      - run:
+          name: list all available browsers
```

Comment:
What is this for?

Reply:
Just for informational purposes. For some reason, I can't access the list of available browsers on saucelabs on my machine, but it works in the CI. Might also be helpful if someone wants to change the tested browsers without setting up a saucelabs account.